src/sagas/modelSaga.js

```diff
@@ -3,6 +3,9 @@ import { ModelManager } from 'composer-concerto';
 import * as selectors from '../selectors/modelSelectors';
 import * as actions from '../actions/modelActions';
 
+/**
+ * saga to validate model and update model manager
```

Comment:
Maybe something along the lines of `saga which yields to putting the successful model manager into the store and subsequently clears all model manager errors from the store`